pkgs/top-level/python-packages.nix

```diff
@@ -10898,6 +10898,8 @@ self: super: with self; {
 
   sdds = callPackage ../development/python-modules/sdds { };
 
+  sdkmanager = callPackage ../development/python-modules/sdkmanager { };
```

Comment:
If this is not a python library, please remove this line

Reply:
It's a library and will be required for the next version of fdroidserver.